docs/install/helm-install/prerequisites/local.md

```
@@ -0,0 +1,57 @@
+| Auth server | `https://FQDN/.well-known/openid-configuration` |
+| FIDO2       | `https://FQDN/.well-known/fido2-configuration`  |
+| SCIM        | `https://FQDN/.well-known/scim-configuration`   |
+
+## Manual Setup
+
+If you prefer manual setup:
+
+1. Install [Minikube](https://minikube.sigs.k8s.io/docs/start/) or [MicroK8s](https://github.com/canonical/microk8s/tree/master#quickstart)
+2. Install [Helm](https://helm.sh/docs/intro/install/)
+3. Create the namespace:
+   ```bash
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)